dnf5.spec

```diff
@@ -143,6 +144,9 @@ BuildRequires:  bash-completion
 BuildRequires:  cmake
 BuildRequires:  doxygen
 BuildRequires:  gettext
+%if %{with appstream}
+BuildRequires:  pkgconfig(appstream) >= 0.16
```

Comment:
This should be moved inside the `libdnf5-plugin-appstream` package, right? Otherwise the dnf5 itself would depend on appstream.